tests/machine/test_machine.py

```diff
@@ -136,3 +136,18 @@ def test_get_machine_schema_types_return_mtype(mocked_types, mocked_machines):
 def test_get_machines_v1(get_client):
     machines = get_client.get_machines()
     assert machines.shape == (49, 10)
+
+
+def test_get_machines_with_query_params(get_client):
+    query_params = {
+        "_only": ["source", "source_clean", "source_type"],
```

Comment:
Should this paramater value be a string?  I am getting confused by
https://github.com/sightmachine/sightmachine-sdk/blob/1ad222d947253f517272c0d8cfde61954ba44d48/smsdk/smsdk_entities/machine/machine_v1.py#L88

Reply:
The type of `kwargs.get("_only")` was string and I had to evaluate it to it's original type so that I can create a select list.